frontend/scripts/pages/profile-actor.page.js

```diff
@@ -325,15 +328,12 @@ const _init = ()  => {
 
   fetch(actorFactsheetURL)
     .then((response) => {
-      if (response.status === 404) {
-        _showErrorMessage();
-        return null;
-      }
-
-      if (response.status === 200) {
+      if (response.ok) {
         return response.json();
       }
-    })
+      _showErrorMessage(response.statusText);
+    }, (reason) => _showErrorMessage(reason.message)
+    )
     .then((result) => {
       if (!result) return;
```

Comment:
On a second thought, if we use catch, we don't have to early return in the next then.